boofuzz/blocks/request.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, name=None, children=None):
         FuzzableBlock.__init__(self, name=name, request=self)
         Node.__init__(self)
         self.label = name  # node label for graph rendering.
+        self._name = name
```

Comment:
Yes, seems that way.

Reply:
It actually was necessary, but only because I set `id = name` as an alias in Request. I did that to support Nodes now having a string id. But the problem was that `Node.__init__(self)` would set id to None. Which, given the alias, would also set name/_name to None again.

Just pushed: I reversed my two changes in request.py, and now pass the name into the Node initializer: `Node.__init__(self, node_id=name)`